extensions/admin-tool-ext-win/package.json

```diff
@@ -1,61 +1,75 @@
-        "title": "%adminToolExtWin.launchSsmsServerPropertiesDialog%",
-        "category": "AdminToolExtWin"
-      }
-    ],
-    "menus": {
-      "commandPalette": [
-        {
-          "command": "adminToolExtWin.launchSsmsServerPropertiesDialog",
-          "when": "false"
-        }
-      ],
-      "objectExplorer/item/context": [
-        {
-          "command": "adminToolExtWin.launchSsmsServerPropertiesDialog",
-          "when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType == Server",
-          "group": "AdminToolExtWin"
-        }
-      ]
-    },
-    "outputChannels": [
-      "admin-tool-ext-win"
-    ]
-  },
-  "dependencies": {
-    "service-downloader": "github:anthonydresser/service-downloader#0.1.5",
-    "vscode-extension-telemetry": "^0.0.15",
-    "vscode-nls": "^3.2.1"
-  },
-  "devDependencies": {
-    "vscode": "1.0.1"
-  }
-}
+	"name": "admin-tool-ext-win",
+	"displayName": "%adminToolExtWin.displayName%",
+	"description": "%adminToolExtWin.description%",
+	"version": "0.0.1",
+	"publisher": "Microsoft",
+	"preview": true,
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+	"icon": "images/sqlserver.png",
+	"aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
+	"engines": {
+		"vscode": "^1.30.1",
+		"sqlops": "*"
+	},
+	"activationEvents": [
+		"*"
+	],
+	"main": "./out/main",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/azuredatastudio.git"
+	},
+	"extensionDependencies": [
+		"Microsoft.mssql"
+	],
+	"contributes": {
+		"commands": [
+			{
+				"command": "adminToolExtWin.launchSsmsMinPropertiesDialog",
+				"title": "%adminToolExtWin.propertiesMenuItem%",
+				"category": "AdminToolExtWin"
+			},
+			{
+				"command": "adminToolExtWin.launchSsmsMinGswDialog",
+				"title": "%adminToolExtWin.launchGswMenuItem%",
+				"category": "AdminToolExtWin"
+			}
+		],
+		"menus": {
+			"commandPalette": [
+				{
+					"command": "adminToolExtWin.launchSsmsMinPropertiesDialog",
+					"when": "false"
+				},
+				{
+					"command": "adminToolExtWin.launchSsmsMinGswDialog",
+					"when": "false"
+				}
+			],
+			"objectExplorer/item/context": [
+				{
+					"command": "adminToolExtWin.launchSsmsMinGswDialog",
+					"when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType == Database",
+					"group": "z-AdminToolExt@1"
+				},
+				{
+					"command": "adminToolExtWin.launchSsmsMinPropertiesDialog",
+					"when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType =~ /Server|Database|Table|Column|Index|Statistic|View|ServerLevelLogin|ServerLevelServerRole|ServerLevelCredential|ServerLevelServerAudit|ServerLevelServerAuditSpecification|StoredProcedure|ScalarValuedFunction|TableValuedFunction|AggregateFunction|Synonym|Assembly|UserDefinedDataType|UserDefinedType|UserDefinedTableType|Sequence|User|DatabaseRole|ApplicationRole|Schema|SecurityPolicy|SecurityPolicie|ServerLevelLinkedServer/",
```

Comment:
I wonder if we should add some kind of shortcut to reference this node collection easier?  Or is this is just a one-off condition.

Reply:
This should just be a one-off. It is tied to the mapping table I have in main but given that I don't see us adding many more nodes to this I'm just going to leave it as is for now and revisit later if it seems worth the effort. 

---
In reply to: [279865064](https://github.com/Microsoft/azuredatastudio/pull/5267#discussion_r279865064) [](ancestors = 279865064)